patches/minecraft/net/minecraft/entity/Entity.java.patch

```diff
@@ -212,16 +212,35 @@
        }
     }
  
-@@ -2063,6 +2092,7 @@
+@@ -2063,33 +2092,32 @@
  
     @Nullable
     public Entity func_212321_a(DimensionType p_212321_1_) {
-+      if (!net.minecraftforge.common.ForgeHooks.onTravelToDimension(this, p_212321_1_)) return null;
++      return changeDimension(p_212321_1_, func_184102_h().func_71218_a(p_212321_1_).func_85176_s());
++   }
++
++   @Nullable
++   public Entity changeDimension(DimensionType destination, net.minecraftforge.common.util.ITeleporter teleporter) {
```

Comment:
These overloads need entries in the exc

Reply:
I'm sorry but I don't understand this. What's the exc?